lib/utils.js

```diff
@@ -147,13 +147,20 @@ Utils = {
 
   // Take a path to a SCSS file and compiles it to CSS with `node-sass`.
   toCSS: function(scss) {
-    if (!Package['chrisbutler:node-sass']) {
+    let sass;
+    let meteorMinorVersionPosition = 1;
+    let minorVersion = Meteor.release.split('.')[meteorMinorVersionPosition];
+    // since meteor 1.3 we no longer need meteor hacks just use the npm version
+    if (minorVersion >= 3){
+      sass = require('node-sass');
+    } else if (!Package['chrisbutler:node-sass']) {
```

Comment:
Dunno... again I'm on meteor 1.4 so if someone can test on a lower version it would be good. All I know is that this is working for us on 1.4. Tried to account for the other versions. BTW also working fine on galaxy.